src/fidesops/ops/service/privacy_request/request_runner_service.py

```diff
@@ -60,6 +62,30 @@
 logger = get_task_logger(__name__)
 
 
+def get_access_manual_webhook_inputs(
+    db: Session, privacy_request: PrivacyRequest
+) -> Tuple[Dict[str, List[Dict[str, Optional[Any]]]], bool]:
+    """Retrieves manually uploaded data for all AccessManualWebhooks and formats in a way
+    to match automatically retrieved data. Also returns if execution should proceed.
+
+    This data will be uploaded to the user as-is, without filtering.
+    """
+    manual_inputs: Dict[str, List[Dict[str, Optional[Any]]]] = {}
```

Comment:
Manual webhooks aren't currently connected to policies, but I am wondering if I should only run this if there are "access" rules defined on the policy?

Reply:
I think this makes sense to add, will be a one-line addition.  This keeps the privacy request from being placed in a `requires_input` stage if the policy has strictly erasure rules only.